docs/README.md

```diff
@@ -121,3 +124,15 @@ Any breaking update to a vendor dependency requires an update to every external
 Generally, breaking vendor dependency updates won't happen often (at least every time kubernetes/client-go updates, maybe) and all the provisioners can be updated with ease, without requiring explicit approval from their respective OWNERS, unless the change is big enough or they've asked that it be required.
 
 As the contributor of a dependency/library update, you're usually responsible for updating the dependents so travis CI passes, as it shouldn't be harder than a find/replace. Otherwise, if it's decided that you don't need to be responsible, some other solution will be worked out to make sure everything stays in a buildable state.
+
+### Using Persistent Volume annotations
+External provisioners may need to store custom data in Persistent Volume annotations. An annotation shall have the below format:
```

Comment:
it would be good to start recommending this convention now, but it's not backwards compatible and IMO it should not be a requirement, so "shall" is too strong a word